src/components/Snackbar.tsx

```diff
@@ -4,15 +4,15 @@ import Slide from '@material-ui/core/Slide';
 import { TransitionProps } from '@material-ui/core/transitions';
 
 export interface SnackbarProps extends MuiSnackbarProps {
-  anchorOrigin: {
+  anchorOrigin?: {
     horizontal: 'center' | 'left' | 'right';
     vertical: 'bottom' | 'top';
   };
-  autoHideDuration: number;
-  color?: 'success' | 'info' | 'warning' | 'error';
-  direction: 'right' | 'left' | 'up' | 'down';
-  severity: 'success' | 'info' | 'warning' | 'error';
-  variant: 'standard' | 'filled' | 'outlined';
+  autoHideDuration?: number;
+  color?: 'success' | 'info' | 'warning' | 'error' | undefined;
```

Comment:
```suggestion
  color?: 'success' | 'info' | 'warning' | 'error';
```
The optional type `?` is the same as having `| undefined` so no need for that

Reply:
Fixed